### DISCUSSION TITLE
Addon Ideas #6 (Paint/costume Editor Addons)

### DISCUSSION THREAD

**Discussion by SpryzenR** [ScratchAddons/ScratchAddons#6385] [Ideas]
Hello again I'm back with more ideas

### Longer costume name length addon

Make an addon that shows more of the costume's name in the Paint Editor because there is so much space there. If needed you should be able to shift some of the costume options to the bottom row but you will be shifting some to the right side a little for the addon

Vector Mode:
![Screenshot (29)](https://github.com/ScratchAddons/ScratchAddons/assets/99098977/f2ac6a68-8aa7-4440-9b8a-3aa02a2db3a7)

Bitmap Mode:

![Screenshot (31)](https://github.com/ScratchAddons/ScratchAddons/assets/99098977/1cb55cae-3da9-4640-8975-34ca4a76af57)

With "Hide Stage and Sprite Pane" Addon:
![Screenshot (50)](https://github.com/ScratchAddons/ScratchAddons/assets/99098977/fcd61ee6-ca40-42c9-a7c0-1983dd68a2ff)

### Move bitmap/vector tools addon 

This addon will make more space outside of the canvas, organize the options for the costume in one area and it'll place the tools somewhere near all the other options I showed in the image above. (this might be wonky with the other addon)

### Hide bitmap/vector tools addon

This addon will also make more space outside of the canvas, but it instead of moving the tools it hides the bitmap/vector tools (ex: the paint bucket and eraser tools). It'll have similar options and animations as the auto-hiding block palette addon that allows it to show/hide. 

### Drop down menu for bitmap/vector tools addon

This addon will just organize the bitmap/vector tools into a drop down menu. Simple.

### Import some stuff from Scratch 2.0 

Just some examples that I believe would fit well.
-One column vector/bitmap tools
-Color Presets (Maybe place them in the space if none of the previous suggested addons are either on or allowed) 
-Zoom Percentage 
-An Updated 3.0 Tips Window that replaces it's newer Projects Tutorial counterpart with updated tutorials, covers for all 3.0 blocks including extensions.

**Comment by BroJac5246** [comment]
I would love to see the suggestions for the ~~paint~~ _costume_ (why!?! It makes no sense!) editor be part of a "Better Costume Editor" addon. Most of the tweaks are small, but they would definitely be useful!

**Reply by WorldLanguages** [reply]
There's also changing the default between bitmap and vector too

**Reply by SpryzenR** [reply]
> There's also changing the default between bitmap and vector too

True, that would be useful for bitmap users.
